### PR TITLE
Add FIO bundle to Konflux

### PR DESCRIPTION
**Pull request by yuumasato**

### CODE REVIEW

**Comment by yuumasato** [comment]
e2e timeout for node checking was merged.
/retest-required

File: bundle-hack/update_csv.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    m['spec']['icon'] = icons
+    print(f"Successfully updated the operator image to use icon in {icon_path}.")
+
+
+def replace_images(m: dict) -> None:
+    # Get operator build image, which is available through the environment
+    # variable as a JSON string. This means we don't have to query brew
+    # directly for data about builds.
+
+    FIO_IMAGE_PULLSPEC = "quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator@sha256:ffd07e3abe5b0c3f863d072a77a81fced02df355e906bca71d2cdc2f8f907850"
```
Comment (rhmdnd) [inline bundle-hack/update_csv.py:115]: Does this need to be updated periodically?

**Review by rhmdnd — APPROVED** [review]
/lgtm

File: bundle-hack/update_csv.py (hunk 1 above)
Comment (yuumasato) [reply bundle-hack/update_csv.py:115]: Konflux automation will keep this updated, it will create PRs to update this file.

**Pull request merged**